recipes/mnnpy/meta.yaml

```diff
@@ -1,43 +1,45 @@
-{% set name = "mnnpy" %}
 {% set version = "0.1.9.5" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: mnnpy
+  version: {{ version }}
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: "41cdafaf21392144b683d7acf99f7a1fc59085f9b3697cd6ace57f40dddace9c"
+  url: https://pypi.io/packages/source/m/mnnpy/mnnpy-{{ version }}.tar.gz
+  sha256: 41cdafaf21392144b683d7acf99f7a1fc59085f9b3697cd6ace57f40dddace9c
 
 build:
-  number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  number: 1
+  skip: True  # [py<34]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - llvm-openmp # [osx]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
```

Comment:
This isn't terribly important, but I'd recommend it as per http://conda-forge.org/docs/maintainer/knowledge_base.html#openmp .